examples/cubic_planar.js

```diff
@@ -94,8 +94,7 @@ for (index = 0; index < wmsLayers.length; index++) {
 
     view.addLayer(tileLayer);
 
-    view.addLayer({
-        update: itowns.updateLayeredMaterialNodeImagery,
+    tileLayer.addColorLayer({
```

Comment:
I clearly prefer explicit (`addColorLayer`) method names than generic ones and this PR moves the API in this direction.

And yes the `type` attribute will become a read-only property (rather than a user defined property)

Reply:
Before there  was already `addImageryLayer`, `addElevationLayer` and `addFeatureLayer`.
What was the motivation to change in `addLayer` and what's new to change the name again.
Because these changes impact the work of users

Reply:
> Before there was already ...

Possibly yes.

> What was the motivation to change in addLayer

The goal was to make itowns different from a globe renderer.
And at the time, adding a generic `addLayer` look like a good idea.

>  Because these changes impact the work of users

Sure. But cumbersome APIs "impact the work of users" as well.

Since we introduced `addLayer` we built more apps using iTowns and in the process realized that the API could be improved, which is the goal of this PR.

Reply:
Ok, technically is there a reason that requires to rename or move function?

Reply:
We could keep the generic `addLayer` name but IMHO it improves the API to have names that match the type of layers you can add.

Reply:
What annoys me going back and forth in the API, which impacts the users.
Especially since previously the names of the API were clear